examples/simple_ipdb.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
Sure, both actually work fine, just the IPDB one has more features, which are shown in the vlan_learning example. For the simple case, using ipr is preferred and less likely to lock up.